### PR TITLE
Add mesh support for sending close session command to all Concorda's clients. See #12

### PR DESCRIPTION
**Pull request by mirceaalexandru**

### CODE REVIEW

File: client/containers/editUser.js [hunk 1]
```diff
@@ -31,20 +31,6 @@ export const EditUser = React.createClass({
     dispatch(upsertUser(userId, data))
   },
```
Comment (mcdonnelldean) [inline client/containers/editUser.js:33]: Why did you remove all this?

File: server/concorda.js [hunk 2]
```diff
@@ -99,7 +101,15 @@ module.exports = function (opts) {
           return response(err)
         }
 
-        response(null, {err: false, sessions: logins.length})
+        // now I should notify all other apps to logout that user
+        that.act('role: concordaclient, cmd: closeSession', {user_id: user_id}, function(err, data){
```
Comment (mcdonnelldean) [inline server/concorda.js:105]: Role should be `role: concorda`, also change cmd to `info: logout`

Comment (mcdonnelldean) [inline server/concorda.js:105]: This can be fire and forget, no need to handle the error or have a callback

File: server/concorda.js [hunk 3]
```diff
@@ -131,6 +141,10 @@ module.exports = function (opts) {
     }
   })
 
+  seneca
+    .use('mesh',{base:true})
```
Comment (mcdonnelldean) [inline server/concorda.js:145]: I'll show you how to remove this tomorrow and fix your timeout issue :dancer:

**Comment by mcdonnelldean** [comment]
@mirceaalexandru Awesome stuff, looks tidy, some comments before the merge.

File: client/containers/editUser.js (hunk 1 above)
Comment (mirceaalexandru) [inline client/containers/editUser.js:33]: I will check again, was part of merging the master branch.
